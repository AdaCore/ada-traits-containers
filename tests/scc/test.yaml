title: 'scc'
description: 'Strongly Connected Components'
driver: 'build_and_exec'
