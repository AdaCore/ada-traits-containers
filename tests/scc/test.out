Found 4 components
Component for A is 1
Component for B is 1
Component for C is 1
Component for D is 3
Component for E is 3
Component for F is 2
Component for G is 2
Component for H is 4
